packages/arb-token-bridge-ui/package.json

```diff
@@ -49,8 +49,9 @@
     "zustand": "^4.3.9"
   },
   "scripts": {
+    "predev": "yarn generateDenylist && yarn generateOpenGraphImages",
     "dev": "next dev",
-    "prebuild": "yarn generateDenylist",
+    "prebuild": "yarn generateDenylist && yarn generateOpenGraphImages",
```

Comment:
Tip: We could've used NextJs's out-of-the-box solution for generating og-images programmatically without the need of prebuilding and setting up folders, if we were using the app-router - https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image#generate-images-using-code-js-ts-tsx

We should keep it for a future iteration.

Reply:
yea we discussed migrating to app/ for various reasons recently (within 1-2 months) and the conclusion is to not migrate in the foreseeable future due to complexity and risk